src/styles/antd-overrides/menu.scss

```diff
@@ -7,6 +7,10 @@
   background: unset !important;
 }
 
+.ant-menu-inline .ant-menu-item {
+  width: auto !important;
```

Comment:
**question** is there a way we can avoid `!important`? Greater specificity perhaps?

Reply:
I tried without it, I can double-check. I remember at the time the antd styles were pretty specific.

Reply:
following up, we can avoid `!important`. ty for the catch.